tests/test-pow.c

```diff
@@ -112,6 +112,12 @@ int main()
         doThePoW(PoW_Context_ptr, pow_ctx);
         int8_t *ret_trytes = getPoWResult(PoW_Context_ptr, pow_ctx);
         assert(ret_trytes);
+#if defined(ENABLE_STAT)
+        PoW_Info *info = (PoW_Info *) getPoWInfo(PoW_Context_ptr, pow_ctx);
+        printf("Hash count: %"PRIu64"\n", info->hash_count);
+        printf("PoW execution time: %.0lf sec\n", info->time);
+        printf("Hash rate: %.3lf per sec\n", info->hash_count / info->time);
```

Comment:
Use the "kH/sec".

Reply:
According to [Wikipedia](https://en.wikipedia.org/wiki/Kilo-), Kilo is a decimal unit prefix in the metric system denoting multiplication by one thousand where it has the unit symbol k, in lower case.